samples/drivers/uart/single_wire/app.overlay

```diff
@@ -0,0 +1,13 @@
+/*
```

Comment:
use a board overlay, not app.overlay so the sample can be re-used for other boards in the future.

Reply:
done